src/router/storeSync/abstractParamConfig.class.js

```diff
@@ -27,16 +27,19 @@ export default class AbstractParamConfig {
      *   added to the URL even though its value is set to the default value of the param.
      * @param {NumberConstructor | StringConstructor | BooleanConstructor, ObjectConstructor} valueType
      * @param {Boolean | Number | String | null} defaultValue
+     * @param {Function} afterSetValuesInStore A function that will be called after the store values
+     *   have been set
      */
     constructor({
         urlParamName,
         mutationsToWatch,
         setValuesInStore,
-        extractValueFromStore,
+        extractValueFromStore = (_) => '',
```

Comment:
might be better to use the default value, meaning `null` as return value here

Reply:
I meant something like `extractValueFromStore: () => null,`

Reply:
i wonder if it is necessary, i think just making the function undefined should work also